EIPS/eip-3675.md

```diff
@@ -98,6 +97,8 @@ Beginning with `TRANSITION_BLOCK`, the block validity conditions **MUST** be alt
 
 *Note*: Validity rules that are not specified in the list above **MUST** remain unchanged.
 
+*Note*: the value of `mixHash` is purposefully non-validated here to provide flexibility for future EIPs.
```

Comment:
Future implies that it might be a while from now. Marginally prefer "subsequent"

```suggestion
*Note*: the value of `mixHash` is purposefully non-validated here to provide flexibility for subsequent EIPs.
```